relay-server/src/actors/outcome.rs

```diff
@@ -357,6 +362,13 @@ mod kafka {
         }
     }
 
+    impl Handler<OutcomePayload> for OutcomeProducer {
+        type Result = Result<(), OutcomeError>;
+        fn handle(&mut self, _message: OutcomePayload, _ctx: &mut Self::Context) -> Self::Result {
+            unimplemented!("TODO implement!")
```

Comment:
Are you planning to implement this before merging or in a follow-up? 

We can get this PR merged even without an implementation here, but in that case it would be better to emit a debug log rather than raise a panic.

Reply:
yes, makes sense, will change to error logs